src/Record/Record.Model/Records.csproj

```diff
@@ -12,7 +12,7 @@
 		<EnvironmentSuffix></EnvironmentSuffix>
 		<ReleaseType></ReleaseType>
 		<PackageId>Record</PackageId>
-		<VersionPrefix>4.0.0$(ReleaseType)</VersionPrefix>
+		<VersionPrefix>4.0.1$(ReleaseType)</VersionPrefix>
```

Comment:
Good catch! We talked about this being a minor, but I agree it's actually a patch.